include/zephyr/bluetooth/conn.h

```diff
@@ -515,6 +515,53 @@ struct bt_conn_le_tx_power_report {
 	int8_t delta;
 };
 
+/** @brief Path Loss zone that has been entered.
+ *
+ *  The path loss zone that has been entered in the most recent LE Path Loss Monitoring
+ *  Threshold Change event as documented in Core Spec. Version 5.4 Vol.4, Part E, 7.7.65.32.
+ *
+ *  @note BT_CONN_LE_PATH_LOSS_ZONE_UNAVAILABLE has been added to notify when path loss becomes
+ *        unavailable.
+ */
+enum bt_conn_le_path_loss_zone {
+	/* Low path loss zone entered. */
+	BT_CONN_LE_PATH_LOSS_ZONE_ENTERED_LOW = 0,
+	/* Middle path loss zone entered. */
+	BT_CONN_LE_PATH_LOSS_ZONE_ENTERED_MIDDLE = 1,
+	/* High path loss zone entered. */
+	BT_CONN_LE_PATH_LOSS_ZONE_ENTERED_HIGH = 2,
+	/* Path loss has become unavailable. */
+	BT_CONN_LE_PATH_LOSS_ZONE_UNAVAILABLE = 3,
+};
+
+/** @brief LE Path Loss Monitoring Threshold Change Report Structure. */
+struct bt_conn_le_path_loss_threshold_report {
+
+	/** Path Loss zone as documented in Core Spec. Version 5.4 Vol.4, Part E, 7.7.65.32. */
+	enum bt_conn_le_path_loss_zone zone;
+
+	/**Current path loss in dBm */
+	uint8_t path_loss_dbm;
+};
+
+/** @brief LE Path Loss Monitoring Parameters Structure as defined in Core Spec. Version 5.4
+ *         Vol.4, Part E, 7.8.119 LE Set Path Loss Reporting Parameters command.
+ */
+struct bt_conn_le_path_loss_reporting_param {
+	/* High threshold for the path loss (dB). */
+	uint8_t high_threshold;
+	/* Hysteresis value for the high threshold (dB). */
+	uint8_t high_hysteresis;
+	/* Low threshold for the path loss (dB). */
+	uint8_t low_threshold;
+	/* Hysteresis value for the low threshold (dB). */
+	uint8_t low_hysteresis;
+	/* Minimum time in number of connection events to be observed once the
+	 * path loss crosses the threshold before an event is generated.
+	 */
```

Comment:
You need to use either `/**` or `/** @brief` for doxygen to pick it up
```suggestion
	/** High threshold for the path loss (dB). */
	uint8_t high_threshold;
	/** Hysteresis value for the high threshold (dB). */
	uint8_t high_hysteresis;
	/** Low threshold for the path loss (dB). */
	uint8_t low_threshold;
	/** Hysteresis value for the low threshold (dB). */
	uint8_t low_hysteresis;
	/** Minimum time in number of connection events to be observed once the
	 *  path loss crosses the threshold before an event is generated.
	 */
```